src/scripts/index.js

```diff
@@ -1,10 +1,224 @@
+ * Call the function which adds a new column after the selected column
+ *        with the name from the user
+*/
+function addColumnToRight() {
+  popup.style.display = 'none';
+  const name = document.getElementById('columnName').value;
+  const clickedColumn = document.getElementById('clickedColumn');
+  event.preventDefault();
+  addTheNewColumn(name, clickedColumn);
+}
+
+
+/**
+ * Create a new Column and added after the 'element'
+ * @param {String} title - the title of the new column
+ * @param {HTMLElement} element - the column after which it has to insert
+ */
+function addTheNewColumn(title, element) {
+  const col = document.getElementById('clickedColumn');
+  col.removeAttribute('id');
+
+  const flexColumn = document.createElement('div');
+  const header = document.createElement('div');
+  const nameColumn = document.createElement('div');
+  const addColumn = document.createElement('div');
+  const editColumn = document.createElement('div');
+  const contentColumn = document.createElement('div');
+
+  header.classList.add('header');
+  flexColumn.classList.add('flex-column');
+  nameColumn.classList.add('name-column');
+  addColumn.classList.add('add-column');
+  editColumn.classList.add('edit-column');
+  contentColumn.classList.add('content-column');
+
+  const addFont = document.createElement('i');
+  const dotsFont = document.createElement('i');
+  addFont.classList.add('fas', 'fa-plus');
+  dotsFont.classList.add('fas', 'fa-ellipsis-h');
+
+  nameColumn.innerHTML = title;
+  addColumn.appendChild(addFont);
+  editColumn.appendChild(dotsFont);
+
+  header.appendChild(nameColumn);
+  header.appendChild(editColumn);
+  header.appendChild(addColumn);
+  flexColumn.appendChild(header);
+  flexColumn.appendChild(contentColumn);
+  header.addEventListener('mouseenter', enterHeader);
+  header.addEventListener('mouseleave', leaveHeader);
+  element.after(flexColumn);
+  event.preventDefault();
+}
+
+
+// tichete
+const popupTicket = document.getElementById('ticket-popup');
+const saveTicket = document.querySelector('#saveTicket');
+const closeTicket = document.getElementById('closeTicket');
+
+/**
+ * Open the popup for adding new ticket
+ */
+function openPopupTicket() {
+  popupTicket.style.display = 'block';
+}
+
+/**
+ * Function for closing the popup corresponding to the ticket
+ */
+function closePopupTicket() {
+  popupTicket.style.display = 'none';
+}
+
+const todoColumn = document.getElementById('idTODO');
+todoColumn.addEventListener('dblclick', openPopupTicket);
+saveTicket.addEventListener('click', addTicket);
+closeTicket.addEventListener('click', closePopupTicket);
+
+/**
+ * Make the popup invisible when pressing the 'add ticket' button
+ * Create the ticket and add to the 'TO DO' column
+ */
+function addTicket() {
+  popupTicket.style.display = 'none';
+  const todoColumn = document.getElementById('contentTODO');
+  const chenar = document.createElement('div');
+  chenar.classList.add('chenar-ticket');
```

Comment:
usually we should stick to a language with regards to classes/variable names etc. As such, please use english for all of these.